### PR TITLE
Installing cross browsers

### PR DESCRIPTION
**Pull request by kaadhina**
Installing chrome and firefox, and dismissing the initial prompts, to enable users to do cross browser testing on hosted agents.

Testing done:
Validated E2E by creating the image, spinning a VM, setting up a build agent and running a build definition to run selenium tests on all browers. Here is a [link](https://mseng.visualstudio.com/VSOnline/_build/index?buildId=5065539&_a=summary&tab=ms.vss-test-web.test-result-details)

### CODE REVIEW

File: images/win/scripts/Installers/Validate-Chrome.ps1 [hunk 1]
```diff
@@ -0,0 +1,17 @@
+################################################################################
+##  File:  Validate-Chrome.ps1
+##  Team:  Automated Testing
+##  Desc:  Validate Google Chrome installation.
+################################################################################
+
+if(Test-Path "HKLM:SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths\chrome.exe")
+{
+	$path = Get-ChildItem "HKLM:SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths\chrome.exe" | Get-ItemProperty -Name Path
+	Write-Host "Google Chrome installed at " $path
```
Comment (chrisrpatterson) [inline images/win/scripts/Installers/Validate-Chrome.ps1]: Please output the version of Chrome in addition to the location

File: images/win/scripts/Installers/Validate-Firefox.ps1 [hunk 2]
```diff
@@ -0,0 +1,17 @@
+################################################################################
+##  File:  Validate-Firefox.ps1
+##  Team:  Automated Testing
+##  Desc:  Validate Mozilla Firefox installation.
+################################################################################
+
+if(Test-Path "HKLM:SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths\firefox.exe")
+{
+	$path = (Get-ItemProperty -Path "HKLM:SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths\firefox.exe").Path
+	Write-Host "Mozilla Firefox installed at " $path
```
Comment (chrisrpatterson) [inline images/win/scripts/Installers/Validate-Firefox.ps1]: Please output the version of FireFox in addition to the location

**Comment by bryanmacfarlane** [comment]
Thanks for spinning up a VM and testing!  Nice to see other teams able to effectively validate :)

File: images/win/scripts/Installers/Validate-Chrome.ps1 (hunk 1 above)
Comment (kaadhina) [reply images/win/scripts/Installers/Validate-Chrome.ps1]: Done. Now displaying both version and location.

File: images/win/scripts/Installers/Validate-Firefox.ps1 (hunk 2 above)
Comment (kaadhina) [reply images/win/scripts/Installers/Validate-Firefox.ps1]: Done. Now displaying both version and location.

**Comment by chrisrpatterson** [comment]
@kaadhina can you resolve the merge conflicts and then merge this PR.  After that can you follow up with a PR to add the documentation generation in the same pattern as https://github.com/Microsoft/vsts-image-generation/blob/f9c0ded8c0f8b124c0e8dd8a9f24b1d1df2895b2/images/win/scripts/Installers/Update-AndroidSDK.ps1#L83.

**Comment by kaadhina** [comment]
@chrisrpatterson I've resolved the conflicts. I do not have permissions to merge though.
Yes, I will follow up with another PR for the documentation generation.

**Pull request merged**